src/api/qualicharge/models/static.py

```diff
@@ -140,3 +141,15 @@ class Statique(ModelSchemaMixin, BaseModel):
     observations: Optional[str]
     date_maj: PastDate
     cable_t2_attache: Optional[bool]
+
+    @model_validator(mode="after")
+    def check_afirev_prefix(self) -> Self:
+        """Check that id_pdc_itinerance and id_station_itinerance prefixes match."""
+        if self.id_station_itinerance[:5] != self.id_station_itinerance[:5]:
```

Comment:
Copy/pasta fail :sweat_smile: 

```suggestion
        if self.id_pdc_itinerance[:5] != self.id_station_itinerance[:5]:
```